bots/local_monitor_bot.py

```diff
@@ -14,15 +14,19 @@
         '所属',
         '良好',
         '優良']
+LOUD_ALARM_FILE = './resources/structure_warning.wav'
 
 logger = logging.getLogger('local-monitor-bots')
 logger.setLevel(logging.INFO)
 
 
 class LocalMonitorBot:
-    def __init__(self, monitor_character, process_id):
+    def __init__(self, monitor_character, process_id, use_quiet_alarm):
         self.monitor_character = monitor_character
         self.pid = process_id
+        self.use_quiet_alarm = use_quiet_alarm
+        self.alarm_cycle_count = 30 if use_quiet_alarm else 10 
```

Comment:
should consider a different variable name. this is the max number of consecutive alarms. also, is the intent to have more alert sounds when quiet_alarm is set? or the other way around.